docs/feature_comparison.md

```diff
@@ -0,0 +1,100 @@
+Feature comparison
+==================
+
+CodeChecker includes two user interfaces, which support different actions. This
+comparison summary is intended to overview which feature is available through
+which interface.
+
+Table of Contents
+-----------------
+
+ * [Analysis invocation](#analysis-invocation)
+ * [Storage of reports to a server](#storage-of-reports-to-a-server)
+ * [Report navigation and visualisation](#report-navigation-and-visualisation)
+    * [Report management, triaging tools](#report-management-triaging-tools)
+    * [Statistics, summary views](#statistics-summary-views)
+ * [Run management](#run-management)
+ * [Server administration](#server-administration)
+    * [Starting a server](#starting-a-server)
+    * [Product management](#product-management)
+    * [Authentication and access control](#authentication-and-access-control)
+
+## Analysis invocation
+
+Analyzers can only be invoked through the command-line command,
+[`analyze`](/docs/user_guide.md#analyze).
+
+Analysis runs locally on the user's computer.
+
+Cross&ndash;Translation-Unit analysis supported if using a [Clang version
+which supports it](http://github.com/Ericsson/clang).
+
+## Storage of reports to a server
+
+Storage of reports can only be done through the command-line command,
+[`store`](/docs/user_guide.md#store).
+
+## Report navigation and visualisation
+
+| Feature name | [Command-line](/docs/user_guide.md#cmd) | Web GUI |
```

Comment:
We should create a link on the `Web GUI` text with the following location: https://github.com/Ericsson/codechecker/blob/master/www/userguide/userguide.md

Reply:
Done. I also forgot this guide is available on GitHub in a valid format. :grinning: